source/extensions/filters/http/dynamo/dynamo_stats.cc

```diff
@@ -0,0 +1,117 @@
+#include "extensions/filters/http/dynamo/dynamo_stats.h"
+
+#include <iostream>
+#include <memory>
+#include <string>
+
+#include "envoy/stats/scope.h"
+
+#include "common/stats/symbol_table_impl.h"
+
+#include "extensions/filters/http/dynamo/dynamo_request_parser.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Dynamo {
+
+DynamoStats::DynamoStats(Stats::Scope& scope, const std::string& prefix)
+    : scope_(scope), pool_(scope.symbolTable()), prefix_(pool_.add(prefix + "dynamodb")),
+      batch_failure_unprocessed_keys_(pool_.add("BatchFailureUnprocessedKeys")),
+      capacity_(pool_.add("capacity")), empty_response_body_(pool_.add("empty_response_body")),
+      error_(pool_.add("error")), invalid_req_body_(pool_.add("invalid_req_body")),
+      invalid_resp_body_(pool_.add("invalid_resp_body")),
+      multiple_tables_(pool_.add("multiple_tables")), no_table_(pool_.add("no_table")),
+      operation_missing_(pool_.add("operation_missing")), table_(pool_.add("table")),
+      table_missing_(pool_.add("table_missing")), upstream_rq_time_(pool_.add("upstream_rq_time")),
+      upstream_rq_total_(pool_.add("upstream_rq_total")) {
+  upstream_rq_total_groups_[0] = pool_.add("upstream_rq_total_unknown");
+  upstream_rq_time_groups_[0] = pool_.add("upstream_rq_time_unknown");
+  for (int i = 1; i < 6; ++i) {
```

Comment:
nit: comment magic 6 or use a named constant?

Reply:
I made such a constant but didn't use it. using now. good catch.